test/parallel/test-eslint-prefer-assert-iferror.js

```diff
@@ -15,11 +15,13 @@ new RuleTester().run('prefer-assert-iferror', rule, {
   invalid: [
     {
       code: 'if (err) throw err;',
-      errors: [{ message: 'Use assert.ifError(err) instead.' }]
+      errors: [{ message: 'Use assert.ifError(err) instead.' }],
+      output: 'assert.ifError(err);'
```

Comment:
I'm guessing this won't work without having `require('assert');`. Same thing below.

Reply:
@shobhitchittora would you be so kind and have a look at this comment?

Reply:
The test would actually work without the `require('assert')` specified. While using assert the user has to require assert to use it right. Also in every other test for rules, this is what is followed.

Reply:
As far as I can tell @apapirovski is right about his comment. I am going to run a CI anyway, so we do not have to speculate on it.